pkg/test/framework/components/ambient/waypoint.go

```diff
@@ -117,7 +117,7 @@ func NewWaypointProxy(ctx resource.Context, ns namespace.Instance, sa string) (W
 	}
 
 	cls := ctx.Clusters().Kube().Default()
-	// Find the Prometheus pod and service, and start forwarding a local port.
+	// Find the Waypoint pod and service, and start forwarding a local port.
 	fetchFn := testKube.NewSinglePodFetch(cls, ns.Name(), fmt.Sprintf("%s=%s", constants.GatewayNameLabel, sa))
 	pods, err := testKube.WaitUntilPodsAreReady(fetchFn)
```

Comment:
@hanxiaop I have not update the framework here to make use of --wait-ready for two minor reasons:
- consistency with what is done in pkg/test/framework/components/istio/kube.go when obtaining the istiod pod
- the current fetcher has a 10 minutes timeout by default and --wait-ready would end up having a stricter timeout.

Anyway I do not have a strong preference for including or omitting the --wait-ready flag

Reply:
Makes sense to me.